webhook/README.rst

```diff
@@ -0,0 +1,143 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======
+Webhook
+=======
+
+Module to receive `webhook <https://en.wikipedia.org/wiki/Webhook/>`_ events.
```

Comment:
Sorry, the URL is `https://en.wikipedia.org/wiki/Webhook` (without the trailing slash).